### PR TITLE
Settlement period ending time & dispute fixes

### PR DESCRIPTION
**Pull request by delfipolito**
- Settlement period: end date fix
- "Drafting jury" quote if vote status is disputed, and "Rulling executed" otherwise
- Dispute status info  change on colors 

image

### CODE REVIEW

File: src/components/Proposals/ProposalDetails/DisputableActionStatus.js [hunk 1]
```diff
@@ -34,7 +36,11 @@ function DisputableActionStatus({ vote }) {
         {challengeEndDate !== 0 && (
           <Item heading="Settlement period">
             <DisputablePeriod
-              endDate={challengeEndDate}
+              endDate={
+                toMs(vote.settledAt) > 0
+                  ? toMs(vote.settledAt)
+                  : challengeEndDate
```
Comment (andy-hook) [inline src/components/Proposals/ProposalDetails/DisputableActionStatus.js:42]: For consistency let's format `settledAt` before using it in the template

File: src/components/Proposals/ProposalDetails/StatusInfo.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ function StatusInfo({ vote }) {
 
   const infoDescription = {
     [VOTE_STATUS_DISPUTED]: (
-      <InfoBox>
+      <InfoBox background="#FFEAEA" borderColor="#FF7C7C" color="#FF7C7C">
```
Comment (andy-hook) [inline src/components/Proposals/ProposalDetails/StatusInfo.js:17]: Can we use the `error` mode instead?

File: src/components/Proposals/ProposalDetails/DisputableActionStatus.js [hunk 3]
```diff
@@ -13,6 +14,7 @@ import { networkEnvironment } from '../../../current-environment'
 
 function DisputableActionStatus({ vote }) {
   const theme = useTheme()
+  console.log(vote)
```
[the remark this answers is not in the document]
Comment (delfipolito) [reply src/components/Proposals/ProposalDetails/DisputableActionStatus.js:17]: ohh, i didnt suppose to push this haha

File: src/components/Proposals/ProposalDetails/StatusInfo.js (hunk 2 above)
Comment (delfipolito) [reply src/components/Proposals/ProposalDetails/StatusInfo.js:17]: Mmmhh, I dont think so. Error colors are diifferent. These are the ones that Paty handled me to put here

Comment (andy-hook) [reply src/components/Proposals/ProposalDetails/StatusInfo.js:17]: We could probably adopt the same colours for the errors in that case, but don't worry about that for now.

How about we use `css={`...`}` for these instead?

**Review by andy-hook — APPROVED** [review]

**Pull request merged**